src/features/cartography/infrastructure/presentation/pages/cartography/cartography.page.ts

```diff
@@ -30,12 +30,18 @@ export class CartographyPage {
 
   public centerView: CenterView = this.cartographyConfiguration;
 
-  public readonly usagerCoordinates$: Observable<Coordinates> = merge(
+  public hasAddressError: boolean = false;
+
+  public readonly usagerCoordinates$: Observable<Coordinates | null> = merge(
```

Comment:
```suggestion
  public readonly usagerCoordinates$: Observable<Coordinates> = merge(
```
Le fait d'ajouter `null` oblige à gérer le cas, il faudrait voir comment s'en passer.

Reply:
le catchError oblige à émettre une valeur, si j'emet une coordinate valide ça me met le point vert donc je ne voit pas comment faire autrement pour l'instant